src/api/app/models/event/workflow_run_fail.rb

```diff
@@ -4,6 +4,7 @@ class WorkflowRunFail < Base
     payload_keys :id, :token_id, :hook_event, :summary, :repository_full_name
 
     receiver_roles :token_executor, :token_member
```

Comment:
```suggestion
    receiver_roles :token_executor, :member
```

Reply:
@ncounter I think the `receiver_role` needs to be adapted as well right?

Reply:
Not really, not in this case. I mean, to keep it clear and more transparent, I set the
`delegate :members, to: :token, prefix: true` with `prefix: true`, so it creates the `token_members` hook instead of just `members`. I think it highlights better the _delegate connection_

Reply:
@ncounter ah sorry, I missed the `prefix` part...